helpers.py

```diff
@@ -2,6 +2,14 @@
 
 from selenium import webdriver
 
+# def page_scroll(driver):
+#     driver.execute_script("window.scrollTo(0, 50);")
+#     time.sleep(5)
+
+# def page_reload(driver):
+#     driver.refresh()
+#     time.sleep(10)
```

Comment:
I do not think that function `page_reload`will be used in future. Now `driver.refresh()` is used everywhere. I think we can delete function `page_reload`